content/en/docs/architecture/step-registry.md

```diff
@@ -175,6 +175,28 @@ ref:
 By default, only secrets in the `test-credentials` namespace will be available for mounting into test steps. Please follow the secret-mirroring
 [documentation](https://github.com/openshift/release/blob/master/core-services/secret-mirroring/README.md) to set up a custom secret in that namespace**
 
+##### Custom Bitwarden items
+
+Adding a new Bitwarden item to create a new secret or inject it in an existing one, requires few minor steps.
```

Comment:
I think this needs a small intro paragraph before we start talking about bitwarden (if I'm oblivious to bitwarden, mentioning it right away without any explanation is not of much help.

Overall, this section feels like it needs a link to some generic "CI Secret Management" document (what is bitwarden, how do you get accoutns, how do you make your secret available for our collection, how they are synced to the cluster), and *this* document should focus only on how to instantiate that process to make a step use that secret.